Rover/Parameters.cpp

```diff
@@ -694,6 +694,12 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @Path: mode_circle.cpp
     AP_SUBGROUPINFO(mode_circle, "CIRC", 57, ParametersG2, ModeCircle),
 
+    // @Param: DRIVE_OPTIONS
+    // @DisplayName: Drive Options
+    // @Description: Rnadom behavioural changes for the vehicle
```

Comment:
spelling

Reply:
Fixed, thanks

Reply:
We've moved to a different parameter now